XamarinCommunityToolkit/Effects/EffectIds.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Xamarin.Forms;
+
+[assembly: ResolutionGroupName("Xamarin.CommunityToolkit.Effects")]
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public class EffectIds
```

Comment:
I really don't like having magic strings around an application and prefer to put them in a common file that is filled with constants. We can do it either way but this approach leads slightly less room for error because if a developer were to add a new effect it is a lot more clear how they need to format the name of the new effect.

Reply:
@Axemasta has a good point here, to make everything in a single file and make it easier to maintain. But otherwise isn't too hard to define a name of the effect and isn't something that we will change a lot, isn't like a `key` that can change on new releases.

@AndreiMisiukevich, said that I don't have a strong opinion on that. Sorry to be that person xD